src/main/java/serverutils/ServerUtilitiesCommon.java

```diff
@@ -268,8 +268,11 @@ public void onServerStarted(FMLServerStartedEvent event) {
     }
 
     public void onServerStopping(FMLServerStoppingEvent event) {
+        Universe universe = Universe.get();
```

Comment:
Call it something else, since it can easily be confused with Universe, a little naming separation is probably a good idea for code readability.

Reply:
Sorry, I don't understand. It's an Universe so why can't it be called universe? It's similar to right below:
```
    public void registerTasks() {
        Universe universe = Universe.get();
        universe.scheduleTask(new DecayTask(), world.chunk_claiming);
        universe.scheduleTask(new CleanupTask(), tasks.cleanup.enabled);
        universe.scheduleTask(new BackupTask(), backups.enable_backups);
        if (auto_shutdown.enabled && auto_shutdown.times.length > 0
                && (auto_shutdown.enabled_singleplayer || universe.server.isDedicatedServer())) {
            universe.scheduleTask(new ShutdownTask());
        }
    }
```

Reply:
Problem I have with it is that the Universe is called in the middle of the other universe calls, which means its easy to confuse the two on a quick glance at the code.

Reply:
Fair enough, I also added a little comment to clarify why I did what I did.